chipy_org/dev_utils/management/commands/makedevdata.py

```diff
@@ -0,0 +1,128 @@
+import sys
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from chipy_org.apps import announcements, job_board, meetings, profiles, sponsors, subgroups
+
+
+class Command(BaseCommand):
+    help = "Adds initial data to the site for developement and testing purposes."
+
+    def handle(self, *args, **options):
+
+        if not settings.DEBUG:
+            print(
+                """
+            DEBUG is not set to True.
+            Exiting without making and changes to database.
+            """
+            )
+            sys.exit(0)
+
+        now = datetime.now()
+        future = now + timedelta(days=30)
+        past = now - timedelta(days=30)
+        times = {"past": past, "now": now, "future": future}
+
+        # add user
+        tyler, _ = User.objects.get_or_create(
+            username="tdurden",
+            first_name="tyler",
+            last_name="durden",
+            email="tdurden@paperstreet.com",
+        )
+
+        narrator, _ = User.objects.get_or_create(
+            username="cornilias",
+            first_name="cornilias",
+            last_name="",
+            email="narrator@paperstreet.com",
+        )
+
+        # announcement data'
```

Comment:
I'm not sure how stringent PR reviews should be for typos in comments, but I found one! 😄

Reply:
sorry I'm not seeing it what's the typo?

Reply:
I missed objects in the line SponsorGroup.objects.update_or_create.  Fixed

Reply:
The typo is an apostrophe on line 45.